### PR TITLE
Examples fixed, warning and redirection added

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

Reorganised the JDBC section under the principle "executeUpdate for non-parameterised single SQL commands, most other commands use PreparedStatement". 

Added an example of pre-preparing a Prepared statement.

Moved the section on simple commands out of PreparedStatement into executeUpdate.

Fixed the example for executeUpdate so that they actually got a statement object.

Changed the re-used versions so they didn't mix inline console IO and DB access, moved values to example parameters.

### CODE REVIEW

File: product_docs/docs/jdbc_connector/42.5.1.2/06_executing_sql_commands_with_executeUpdate().mdx [hunk 1]
```diff
@@ -77,7 +85,30 @@ The catch block displays an appropriate error message to the user if the program
     }
 ```
 
-You can use `executeUpdate()` with any SQL command that doesn't return a result set. However, you probably want to use `PrepareStatements` when the queries can be parameterized.
+You can use `executeUpdate()` with any SQL command that doesn't return a result set. It is best suited to situations where a specific command needs to be executed and that command takes no parameters:
```
Comment (drothery-edb) [inline product_docs/docs/jdbc_connector/42.5.1.2/06_executing_sql_commands_with_executeUpdate().mdx:88]:
```suggestion
You can use `executeUpdate()` with any SQL command that doesn't return a result set. It is best suited to situations where a specific command needs to be executed and that command takes no parameters.
```

File: product_docs/docs/jdbc_connector/42.5.1.2/06_executing_sql_commands_with_executeUpdate().mdx [hunk 2]
```diff
@@ -30,41 +34,45 @@ The example that follows shows using the `executeUpdate()` method to add a row t
...
-        System.out.println("Success - "+rowcount+
-          " rows affected.");
-      } catch(Exception err) {
-        System.out.println("An error has occurred.");
-        System.out.println("See full details below.");
-        err.printStackTrace();
-      }
+        try
+        {
+            Statement stmt=con.createStatement();
```
Comment (yeroc) [inline product_docs/docs/jdbc_connector/42.5.1.2/06_executing_sql_commands_with_executeUpdate().mdx:41]: Best practice is to use try-with-resources here: 
eg.
```
    try (Statement stmt = con.createStatement()) 
    {
        ....
```

File: product_docs/docs/jdbc_connector/42.5.1.2/06_executing_sql_commands_with_executeUpdate().mdx [hunk 3]
```diff
@@ -19,7 +19,11 @@ The signature of the `executeUpdate()` method is:
     int executeUpdate(String sqlStatement)
 ```
 
-Provide this method a single parameter of type `String` containing the SQL command that you want to execute.
+Provide this method a single parameter of type `String` containing the SQL command that you want to execute. 
+
+!!!Warning ""
```
Comment (yeroc) [inline product_docs/docs/jdbc_connector/42.5.1.2/06_executing_sql_commands_with_executeUpdate().mdx:24]: 👍

File: product_docs/docs/jdbc_connector/42.5.1.2/06_executing_sql_commands_with_executeUpdate().mdx [hunk 4]
```diff
@@ -86,24 +117,24 @@ Many applications execute the same SQL statement over and over again, changing o
...
-          stmt.execute();   
-          System.out.println("The procedure successfully executed.");    
-          } catch(Exception err) {           
-            System.out.println("An error has occurred.");    
-            System.out.println("See full details below.");  
-            err.printStackTrace();  
-          } 
+    public void addEmployee(Connection con, Integer id, String name)
+    {
+        try {
```
Comment (yeroc) [inline product_docs/docs/jdbc_connector/42.5.1.2/06_executing_sql_commands_with_executeUpdate().mdx:122]: Again, recommend using try-with-resources, it adds no additional lines of code so doesn't impair readability.

File: product_docs/docs/jdbc_connector/42.5.1.2/06_executing_sql_commands_with_executeUpdate().mdx (hunk 2 above)
Comment (djw-m) [reply product_docs/docs/jdbc_connector/42.5.1.2/06_executing_sql_commands_with_executeUpdate().mdx]: Changed

**Pull request closed without merging**